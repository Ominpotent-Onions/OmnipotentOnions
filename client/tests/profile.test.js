import React from 'react';
import Profile from '../src/components/profile';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Profile />).toJSON();
  expect(tree).toMatchSnapshot();
});