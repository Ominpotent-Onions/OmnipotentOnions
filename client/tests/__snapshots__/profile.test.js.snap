// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    The Chosen
  </h1>
  <img
    src="https://upload.wikimedia.org/wikipedia/commons/0/09/Spam_can.png"
  />
</div>
`;
